releasenotes/notes/1.1/classical-store-e64ee1286219a862.yaml

```
@@ -44,8 +44,8 @@ features_circuits:
 
     A variable is a :class:`.Var` node, which can now contain an arbitrary type, and represents a
     unique memory location within its live range when added to a circuit.  These can be constructed
-    in a circuit using :meth:`.QuantumCircuit.add_var` and :meth:`~.QuantumCircuit.add_input`, or
-    at a lower level using :meth:`.Var.new`.
+    in a circuit using :meth:`.QuantumCircuit.add_var` and :meth:`.QuantumCircuit.add_input`, or
+    at a lower level using :func:`~.qiskit.circuit.classical.expr.Var.new`.
```

I think this was correct before. The cross ref is broken looking at the artifact locally.
```suggestion
    at a lower level using :func:`.Var.new`.
```